lib/controllers/flow/account_linking_flow_controller.dart

```diff
@@ -87,6 +87,9 @@ class AccountLinkingFlowController extends GetxController {
       case ConsentStatus.revoked:
```

Comment:
nit:
enums should be all UPPERCASE

Reply:
https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names

enums in dart should be lowercase